### PR TITLE
feat: add multi-select for asset types in agent group creation

### PR DESCRIPTION
**Pull request by Zakaria-Latif**
Add multi-select functionality for asset types in the agent group creation form

[Screencast from 04 شتنبر, 2024 +01 14:42:41.webm](https://github.com/user-attachments/assets/63008ab9-49cf-4d85-8e18-1746e565340d)

### CODE REVIEW

**Review by adnaneserrar — COMMENTED** [review]
Can you a screen shot or video showing the changes in the UI?

**Review by rabsondev — APPROVED** [review]
Can you please add a picture/video in the description.

**Comment by 3asm** [comment]
Please add description and screenshot of the new implementation.

**Review by PiranhaSa — APPROVED** [review]

**Review by nmasdoufi-ol — APPROVED** [review]
Great!

**Review by adnaneserrar — APPROVED** [review]

**Comment by 3asm** [comment]
The drop-down needs some rework. It should have a human-friendly name, and an icon next to it. Also, the spacing with the bottom is missing.

**Review by rabsondev — DISMISSED** [review]

**Review by ohachimOs — APPROVED** [review]

File: project/agent-groups/components/AgentGroupForm.vue [hunk 1]
```diff
@@ -70,6 +70,66 @@
...
+      clearable
+      multiple
+      label="Choose Asset Types"
+      placeholder="Choose Asset Types"
+    >
+      <template #selection="{ item }">
+        <v-chip
+          :key="item.value"
+        >
+          {{ assetTypeTitles[item.title] }}
```
Comment (3asm) [inline project/agent-groups/components/AgentGroupForm.vue:89]: Please add the icon here as well. use outlined chip.

**Review by 3asm — CHANGES_REQUESTED** [review]

**Review by 3asm — APPROVED** [review]

**Pull request merged**